tests/framework/util.go

```diff
@@ -127,3 +127,10 @@ func DeleteServiceAccount(saName string) {
 	err = virtCli.CoreV1().ServiceAccounts(NamespaceTestDefault).Delete(saName, nil)
 	Expect(err).ToNot(HaveOccurred())
 }
+
+func RemoveDV(dvName string) {
+	virtCli, err := kubecli.GetKubevirtClient()
+	Expect(err).ToNot(HaveOccurred())
+	err = virtCli.CdiClient().CdiV1alpha1().DataVolumes(NamespaceTestDefault).Delete(dvName, nil)
```

Comment:
This will only remove from 'NamespaceTestDefault' maybe it's better if the function get the ns.